Build failure: incorrect path in generated vesabios_pm.o_offsets.c

**Describe the bug**

Trying to build, I get this error:
```
cc -c -o vesabios_pm.o vesabios_pm.o_offsets.c
vesabios_pm.o_offsets.c:16:10: fatal error: /home/data/devel/dosemu2/dosemu2/src/home/data/devel/dosemu2/dosemu2/src/base/dev/vga/vesabios_pm_offsets.h: No such file or directory
   16 | #include "/home/data/devel/dosemu2/dosemu2/src/home/data/devel/dosemu2/dosemu2/src/base/dev/vga/vesabios_pm_offsets.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**To Reproduce**

Build on Debian Sid. Maybe the error is due to symlinks in the path to srcdir…


**dosemu2 origins**

git HEAD

Hmm, I have another similar error, starting to think `srcdir` is set wrong by configure.
```
make[2]: *** No rule to make target '/home/data/devel/dosemu2/dosemu2/src/home/data/devel/dosemu2/dosemu2/src/base/bios/x86/bios.S', needed by 'bios1.o'.  Stop.
```

--- a/Makefile.conf.in
+++ b/Makefile.conf.in
@@ -23,7 +23,7 @@ fdtarball:=@fdtarball@
 username := @username@
 cmdsuff:=@cmdsuff@
 abs_top_srcdir:=@abs_top_srcdir@
-abs_top_builddir:=@abs_top_builddir@
+abs_top_builddir:=$(realpath @abs_top_builddir@)
 
 INCDIR=-I${top_builddir}/src/include -I${top_builddir}/src/plugin/include \
     -I${top_srcdir}/src/base/bios/x86 -I${top_srcdir}/src/include \

